iot/dotnet-iot-and-nanoframework/README.md

```diff
@@ -0,0 +1,67 @@
+--- 
+languages:
+- csharp
+products:
+- dotnet
+- dotnet-core
+- azure
+- azure-iot-hub
+page_type: sample
+name: "Compare .NET nanoFramework with .NET Core IoT"
+urlFragment: "compare-nanoframework-with-dotnet-core-iot"
+description: "A sample solution and documentation for comparing .NET nanoFramework and .NET Core IoT platforms. For this documentation same sample application with an identical scenario is created on both platforms, M5Stack with .NET nanoFramework and Raspberry Pi 2 (or above) with .Net Core IoT"
+---
+
+# Compare .NET nanoFramework with .NET Core IoT
+
+## What's .NET nanoFramework
+
+[.NET nanoFramework](https://www.nanoframework.net/) is a free and Open Source platform that enables the writing of managed code applications for constrained embedded devices. It is suitable for many types of projects including IoT sensors, wearables, academic proof of concept, robotics, hobbyist/makers creations or even complex industrial equipment.
+
```

Comment:
You can add a link on the Show Dotnet article: https://devblogs.microsoft.com/dotnet/show-dotnet-running-my-net-nanoframework-for-8-years-on-a-battery/

This gives example of production products running .NET nanoFramework and connected to Azure IoT.

Reply:
Thanks this is added